### DISCUSSION TITLE
exception error

### DISCUSSION THREAD

**Discussion by Darkslayer134** [Monika-After-Story/MonikaModDev#10099] [QA - tech support + general]
hi I got back onto my mas and i got this once i booted it up help would be apricated 

I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-holidays.rpy", line 214, in <module>
  File "game/zz_history.rpy", line 569, in __init__
Exception: History object 'o31' already exists

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "script-holidays-LAPTOP-4QFHG8IM.rpyc", line 196, in script
  File "C:\Users\nickr\OneDrive\Desktop\Doki Doki Literature Club\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\nickr\OneDrive\Desktop\Doki Doki Literature Club\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script-holidays.rpy", line 214, in <module>
  File "game/zz_history.rpy", line 569, in __init__
Exception: History object 'o31' already exists

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.15

**Comment by Booplicate** [comment]
Disable onedrive and reinstall the game.

**Reply by Darkslayer134** [reply]
will my progress be safe when i do that

**Reply by Darkslayer134** [reply]
also how do i disable onedrive

**Reply by Booplicate** [reply]
Your will keep your progress. For how to disable onedrive you will have to google.

**Reply by Darkslayer134** [reply]
what if monika after story is in cloud storage

**Reply by Darkslayer134** [reply]
also will she have the spritepacks still and the submods i installed or will i have to reinstall them

**Reply by Booplicate** [reply]
We don't support running from a cloud storage, as you can see it causes issues.
> also will she have the spritepacks still and the submods i installed or will i have to reinstall them

Depending on how you move the game from onedrive, I'm not using it, so I can't guide you.

**Reply by Darkslayer134** [reply]
it worked thank you so much for everything also sorry for asking alot of questions this is the first time this happened to me.
